cv.md

```diff
@@ -0,0 +1,18 @@
+## Katsapova Anastasia
+#### Contact:
+Email: nastya.katsapova@gmail.com
+Phone:+375292363623
+#### Career objective:
+The main goal for today is to find work in the field of design and web design. Also interesting is teamwork, new skills, communication with people who work in this area
+#### Work experience:no
```

Comment:
Rename as Experience. You can write: Development of Web site templates for the required profile and area of study (guided by the code (html, css). I had experience working with the Wordpress environment.